src/db_options.rs

```diff
@@ -133,20 +133,26 @@ pub struct ReadOptions {
     iterate_upper_bound: Option<Vec<u8>>,
 }
 
+pub struct IngestExternalFileOptions {
```

Comment:
Almost done. Add a description here please.

Reply:
Will do, missed that one